MANIFEST.in

```
@@ -9,7 +9,7 @@ include requirements_docs.txt
 
 recursive-include figanos *.json *.mplstyle *.py *.txt
 recursive-include docs *.rst conf.py Makefile make.bat *.jpg *.png *.gif
-recursive-include data *.json *.txt
+recursive-include data *.json *.txt *.yml
```

```suggestion
recursive-include data *.json *.txt *.yml *.png
```
Shouldn't we want to add the logo too ?

---

Good question. I think we should opt for a different solution. I can probably suggest an implementation tomorrow. See: #115